docker/testing/ubuntu2204_java_driver_testing

```diff
@@ -0,0 +1,56 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:22.04
+MAINTAINER Apache Cassandra <dev@cassandra.apache.org>
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+	apt update  && \
+	apt install -y curl git virtualenv software-properties-common vim maven sudo && \
+	add-apt-repository -y ppa:deadsnakes/ppa && \
+	apt update
+
+# set up python 3.9
+RUN DEBIAN_FRONTEND=nointeractive apt install -y python3.9
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.9 1
+
+# add user
+RUN useradd -rm -d /home/docker -s /bin/bash -g root -G sudo -u 1000 -p "$(openssl passwd -1 docker)" docker
+RUN echo "docker ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+
+USER docker
+
+WORKDIR /home/docker
+
+# install jabba and java versions
+RUN export JABBA_HOME="$HOME/.jabba" && \
+	curl -sL https://github.com/Jabba-Team/jabba/raw/main/install.sh | bash && . /home/docker/.jabba/jabba.sh
+	
+RUN . /home/docker/.jabba/jabba.sh && jabba install openjdk@1.8.0-292
+RUN . /home/docker/.jabba/jabba.sh && jabba install openjdk@1.11.0-9
+RUN . /home/docker/.jabba/jabba.sh && jabba install openjdk@17
+
+ENV PATH="/home/docker/.jabba/bin:/home/docker/.local/bin:$PATH"
+
+RUN echo "export JAVA8_HOME=$(jabba which openjdk@1.8.0-292)" >> ~/env.txt && \
+  	echo "export JAVA11_HOME=$(jabba which openjdk@1.11.0-9)" >> ~/env.txt && \
+	echo "export JAVA17_HOME=$(jabba which openjdk@17)" >> ~/env.txt && \
+  	echo "export JAVA_HOME=$(jabba which openjdk@1.8.0-292)" >> ~/env.txt && \
+  	echo ". $HOME/.jabba/jabba.sh" >> ~/env.txt && \
+  	echo "jabba use openjdk@1.8.0-292" >> ~/env.txt
+
+# ccm
+RUN git clone https://github.com/riptano/ccm.git  && \
+	cd ccm && \
+	pip install -e .
```

Comment:
is this version of ccm then baked into the image? 
how do we test against newer versions of ccm ? what if the driver needs a newer ccm version to pass CI ?
(we don't have to have a fix here, just have awareness of this)

Reply:
Yes, it is baked into the image, same for other drivers' Jenkins AMI images. We just need to rebuild the images later when we want newer ccm.